recipes/envyaml/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+python_min:
```

Comment:
You can delete this file since conda-forge does not really officially support python<3.9 at this time. If you want to keep the lower pin, I would just include it as a jinja variable in the `meta.yaml`:

```
{% set name = "envyaml" %}
{% set version = "1.10.211231" %}
{% set python_min = "3.7" %}
```